src/app/shared/services/children/children.service.ts

```diff
@@ -20,19 +21,31 @@ export class ChildrenService {
     private store: Store,
   ) { }
 
-  private setParams( searchString?:string ): HttpParams {
+  private setParams( parameters?: ChildrenParameters, isParent?: boolean ): HttpParams {
     let params = new HttpParams();
 
-    if(searchString){
-      params = params.set('SearchString', searchString);
+    if(parameters){
+      if(parameters.searchString){
+      params = params.set('SearchString', parameters.searchString);}
+      if(parameters.tabTitle){ 
+        if(parameters.tabTitle == "Батьки"){
+          isParent= true;
+        }
+        if(parameters.tabTitle == "Діти"){
+          isParent= false;
+        }
+      }
+
+    if(isParent !== undefined){
```

Comment:
I'm not sure about this check. I guess isParent should be false by default. @litvinets what do you think?